ghostwriter/reporting/models.py

```diff
@@ -223,7 +237,7 @@ class Finding(models.Model):
     extra_fields = models.JSONField(default=dict)
 
     class Meta:
-        ordering = ["severity", "-cvss_score", "finding_type", "title"]
+        ordering = ["severity", Coalesce("cvss_v4_score", "cvss_score").desc(nulls_last=True), "finding_type", "title"]
```

Comment:
Functions don't work in Django pre-4.1 due to a bug, and indeed this breaks the findings list page.